junebug/channel.py

```diff
@@ -201,6 +201,19 @@ def get_all(cls, redis):
         channels = yield redis.smembers('channels')
         returnValue(channels)
 
+    @classmethod
+    @inlineCallbacks
+    def start_all_channels(cls, redis, config, parent):
+        '''Ensures that all of the stored channels are running'''
+        for id in (yield cls.get_all(redis)):
+            try:
+                yield cls.from_id(redis, config, id, parent)
```

Comment:
Having a look, currently `channel.start` is not a no-op for started channels, it will fail when we try to `setServiceParent` with `cannot have two services with the same name`. 

But I don't think there is a possible case when we have it in memory but not started. We use `disownServiceParent` when stopping, so even if it's in memory, it won't be able to be found.